modules/core/src/main/java/org/apache/ignite/internal/cdc/CdcMain.java

```diff
@@ -203,6 +213,9 @@ public class CdcMain implements Runnable {
     /** Time of last segment consumption. */
     private AtomicLongMetric lastSegmentConsumptionTs;
 
+    /** Time of WAL segment events processing by a CDC consumer, in milliseconds. */
+    private HistogramMetricImpl eventsConsumptionTime;
```

Comment:
events -> evt

Move the declaration closer to `evtCaptureTime` and describe in javadocs the time range (see how `evtCaptureTime` javadoc is written). Do the same for metric description.

Reply:
Done!